google-chart.html

```diff
@@ -384,6 +404,7 @@
             var chart = chartAndData[0];
             var data = chartAndData[1];
             google.visualization.events.addOneTimeListener(chart, 'ready', function() {
+              this._setImageUri(this.getImageURI());
```

Comment:
Not sure we should incur a perf hit on every draw call. Many users will never need the image url. 

How about defining `imageUri` as a getter and caching the value the first time it's called?